src/js/ripple/remote.js

```diff
@@ -1424,10 +1425,19 @@ Remote.prototype.requestAccountTx = function(options, callback) {
   }, options);
 
   request.once('success', function(res) {
-    if (options.parseBinary) {
-      res.transactions = res.transactions.map(Remote.parseBinaryTransaction);
+    if (!options.parseBinary) {
+      request.emit('transactions', res);
+      return;
     }
-    request.emit('transactions', res);
+
+    async.mapSeries(res.transactions, function(transaction, next) {
+      async.setImmediate(function() {
```

Comment:
Would be better to replace with `async.nextTick`. Internally `async.setImmediate` is assigned to `async.nextTick`, but the API docs only show `nextTick` as an available method (https://github.com/caolan/async#nextTick)